R/internal.R

```diff
@@ -74,7 +82,7 @@ rolling_mean_time_date <- function(date_time_vec,window_size) {
   # Convert to numeric
   numeric_dates <- as.numeric(lubridate::dmy_hms(date_time_vec))
   # Calculate the mean
-  mean_numeric <- RcppRoll::roll_mean(numeric_dates,
+  mean_numeric <- RcppRoll::roll_var(numeric_dates,
```

Comment:
I don't get this, but maybe I misunderstood original purpose

Reply:
Good catch, this is a mistake I and will revert!